doc/sphinxext/math_symbol_table.py

```diff
@@ -5,18 +5,19 @@
 
 symbols = [
     ["Lower-case Greek",
-     6,
-     r"""\alpha \beta \gamma \chi \delta \epsilon \eta \iota \kappa
-         \lambda \mu \nu \omega \phi \pi \psi \rho \sigma \tau \theta
-         \upsilon \xi \zeta \digamma \varepsilon \varkappa \varphi
-         \varpi \varrho \varsigma \vartheta""".split()],
+     4,
+     {r"\alpha", r"\beta", r"\gamma",  r"\chi", r"\delta", r"\epsilon",
+      r"\eta", r"\iota",  r"\kappa", r"\lambda", r"\mu", r"\nu",  r"\omega",
+      r"\phi",  r"\pi", r"\psi", r"\rho",  r"\sigma",  r"\tau", r"\theta",
+      r"\upsilon", r"\xi", r"\zeta",  r"\digamma", r"\varepsilon", r"\varkappa",
+      r"\varphi", r"\varpi", r"\varrho", r"\varsigma",  r"\vartheta"}],
     ["Upper-case Greek",
-     8,
-     r"""\Delta \Gamma \Lambda \Omega \Phi \Pi \Psi \Sigma \Theta
-     \Upsilon \Xi \mho \nabla""".split()],
+     4,
+     {r"\Delta", r"\Gamma", r"\Lambda", r"\Omega", r"\Phi", r"\Pi", r"\Psi",
```

Comment:
Not sure that I would have encouraged this type of change for consistency, but when it is here...

Reply:
(Also, this require automatic sorting, which is maybe not a step in the right direction, despite improving it substantially here.)

Reply:
Well, anything I have said regarding that should ideally have been "do not turn it into a set so that it can be manually sorted", but anyway.

I *think* that set is probably not the best thing here. It would admittedly remove duplicates, but that is about it. All other benefits (like fast member checking) should not really be a factor. Strictly, I guess a Tuple may be the best for these "static" ones. Not that it matters, but it is sometimes a bit sensitive to sub-optimize these things, so may just as well change to the best if is it to be changed. I'd be happy with the old form as well though.